Apps/Admin/Client/Store/PatientDetails/PatientSupportDetailsReducers.cs

```diff
@@ -33,27 +33,29 @@ public static MessageVerificationState ReduceLoadAction(MessageVerificationState
         }
 
         [ReducerMethod]
-        public static MessageVerificationState ReduceLoadSuccessAction(MessageVerificationState state, MessageVerificationActions.LoadSuccessAction action)
+        public static PatientSupportDetailsState ReduceLoadSuccessAction(PatientSupportDetailsState state, PatientSupportDetailsActions.LoadSuccessAction action)
         {
-            ImmutableList<MessagingVerificationModel> data = state.Data ?? new List<MessagingVerificationModel>().ToImmutableList();
+            ImmutableList<MessagingVerificationModel> messageVerifications = state.MessagingVerifications ?? new List<MessagingVerificationModel>().ToImmutableList();
 
-            IEnumerable<MessagingVerificationModel>? verifications = action.Data.ResourcePayload;
+            IEnumerable<MessagingVerificationModel>? verifications = action.Data?.MessagingVerifications;
             if (verifications != null)
             {
-                data = data.RemoveAll(x => x.UserProfileId == action.Hdid).AddRange(verifications);
+                messageVerifications = messageVerifications.RemoveAll(x => x.UserProfileId == action.Hdid).AddRange(verifications);
             }
 
             return state with
             {
                 IsLoading = false,
                 Result = action.Data,
                 Error = null,
-                Data = data,
+                MessagingVerifications = messageVerifications,
+                BlockedDataSources = action.Data?.BlockedDataSources,
+                AgentActions = action.Data?.AgentActions,
             };
         }
```

Comment:
This reducer preserves existing messaging verifications for other patients, but doesn't support the same for blocked data sources or agent actions.  Since we perform a new request for patient details on every page load, we may as well make messaging verifications behave the same as the others, so the entire state is concerned with the most recently retrieved patient.

Reply:
I hope I understood your comment fully here. I removed the filtering to be inline with the other details values.

Reply:
Yeah, that's what I was getting at.